client/src/components/manageProjects/editProject.js

```diff
@@ -63,147 +239,151 @@ const EditProject = ({
-        fieldName="hflaWebsiteUrl"
-        updateProject={updateProject}
-        fieldTitle="HfLA Website URL:"
-        accessLevel={userAccessLevel}
-        canEdit={['admin']}
-      />
-      {/* <EditableField
-        fieldData={projectToEdit.videoConferenceLink}
-        fieldName="videoConferenceLink"
-        updateProject={updateProject}
-        fieldTitle="Video Conference Link:"
-        accessLevel={userAccessLevel}
-        canEdit={['admin']}
-      /> */}
-      {/* <EditableField
-        fieldData={projectToEdit.lookingDescription}
-        fieldName="lookingDescription"
-        updateProject={updateProject}
-        fieldTitle="Looking For Description:"
-        accessLevel={userAccessLevel}
-        canEdit={['admin']}
-      /> */}
-      <EditableField
-        fieldData={partnerDataFormatted}
-        fieldName="partners"
-        updateProject={updateProject}
-        fieldTitle="Partners (comma separated):"
-        accessLevel={userAccessLevel}
-        canEdit={['admin']}
-      />
-      {/* <EditableField
-        fieldData={recrutingDataFormatted}
-        fieldName="recruitingCategories"
-        updateProject={updateProject}
-        fieldTitle="Recruiting Categories (comma separated):"
-        accessLevel={userAccessLevel}
-        canEdit={['admin']}
-      /> */}
-      <div className="event-list">
-        <h3>Recurring Events</h3>
-        <h2 className="event-alert">{eventAlert}</h2>
-        <ul>
-          {rEvents.map((event) => (
-            // eslint-disable-next-line no-underscore-dangle
-            <li key={`${event.event_id}`}>
-              <button type="button" onClick={() => setSelectedEvent(event)}>
-                <div>{event.name}</div>
-                <div className="event-list-details">
-                  {`${event.dayOfTheWeek}, ${event.startTime} - ${event.endTime}; ${event.eventType}`}
-                   <div className="edit-icon"><EditIcon /></div>
-                   </div>
-                <div className="event-list-description">{`${event.description}`}</div>
-              </button>
-            </li>
-          ))}
-        </ul>
-      </div>
-      <div className="display-events">
-        <button
-          type="button"
-          className="create-form-button"
-          onClick={() => setIsCreateNew(true)}
+      <Box sx={{ textAlign: 'center' }}>
+        <Typography variant="h1">Project Management</Typography>
+      </Box>
+      <Box sx={{ bgcolor: '#F5F5F5' }}>
+        <Box sx={{ p: 2, display: 'flex', justifyContent: 'space-between' }}>
+          <Box>
+            <Typography sx={{ fontSize: '18px', fontWeight: '600' }}>
+              Project Information
+            </Typography>
+          </Box>
+        </Box>
+        <Divider sx={{ borderColor: 'rgba(0,0,0,1)' }} />
+        <Box sx={{ py: 2, px: 4 }}>
+          <form id="project-form" onSubmit={handleSubmit}>
+            {simpleInputs.map((input, event) => (
```

Comment:
Also would the Patch request be a Put since the entire Project Object is being submitted?

Reply:
Thanks @bkmorgan3 . You are correct, it should be a PUT instead of a PATCH because I am submitting all the data. Good catch.

In regards to 2, I just went with whatever was on the Figma design but I know that is a feature we did talk about once. Wasn't sure if it was decided on so I just applied the UI.

In regards to 3, I am not completely sure. From my understanding only admin and project managers on that specific team (?) are able to view the projects and edit them. I don't think users or PMs not on that team have access to them.

Reply:
1. Yeah, this is good work.

2. I agree we should look for an indication that the project is being edited. I'll ping the design team about that.

@juliagab56 Can you wiegh in on this?

Seems like it should go here - there's a spot for it where the Add Project SVG is:
<img width="398" alt="image" src="https://github.com/hackforla/VRMS/assets/5898009/d92950bc-bccc-4572-b0dd-13da8f5a83b8">

<img width="379" alt="image" src="https://github.com/hackforla/VRMS/assets/5898009/1699b979-54b0-4076-9545-9f46cc64c86f">

Disregard all the little letters in the boxes, that's an extension of mine